doc/man3/OSSL_PARAM_int.pod

```diff
@@ -177,6 +179,9 @@ OSSL_PARAM structure.
 OSSL_PARAM_locate() is a function that searches an B<array> of parameters for
 the one matching the B<key> name.
 
+OSSL_PARAM_locate_const() behaves exactly like OSSL_PARAM_locate() but accepts
+and returns a constant array of parameters.
```

Comment:
It isn't meant to *return* an array...

Reply:
I must remember to engage brain before typing...
Fixed now.